.github/actions/install-languages-cli/action.yml

```diff
@@ -2,6 +2,10 @@ name: Install Languages CLI
 description: Downloads the Languages CLI from a known release or installs using Cargo
 
 inputs:
+  branch:
```

Comment:
Could you add a comment explaining that we have to do this due to `job_workflow_sha` not being available, and link to https://github.com/actions/runner/issues/2417 ?